language/tools/vm-genesis/src/lib.rs

```diff
@@ -388,8 +388,10 @@ fn create_and_initialize_owners_operators(
         let owner_auth_key = AuthenticationKey::ed25519(&owner_key);
         let owner_account = account_address::from_public_key(owner_key);
         let create_owner_script = transaction_builder::encode_create_validator_account_script(
+            0,
             owner_account,
             owner_auth_key.prefix().to_vec(),
+            vec![],
```

Comment:
It would be super nice to try to put some readable name in here, like b"validator_0", b"validator_1", b"validator_2" if it's not too hard (I was trying to do this part, but may should've rolled out it like you did it would've been faster :) )

And same below for operators: b"operator_0", b"operator_1", ...

Reply:
Agree with this and would like to see it here! Ensuring that the `human_name` of each validator/operator is unique seems important to avoid confusion.